### DISCUSSION TITLE
HELP: struggling to figure this out

### DISCUSSION THREAD

**Discussion by th3ch0s3ntw0** [maruohon/litematica#831] [Q&A] [answered]
ive watched a couple youtube videos, but cant seem to find my answer, i was using litematica on forge about a week ago and somehow made a selection using the Simple mode, its just a 1x1 selection on a random piece of grass in my survival server, but i have NO CLUE how to delete it, the selection shows up NOWHERE in the litematica menu and i have no clue how to delete it from my world. although it nots like insanely annoying, its just kind of an eyesore as its right outside my front base. pls lmk how to fix this!!!
![2024-03-28_19 49 15](https://github.com/maruohon/litematica/assets/165350050/992a0c34-051e-4c23-980b-eb0a0aabac72)

**Comment by sakura-ryoko** [comment]
Switch it to normal mode, viola, simple 1x1 box gone.

**Reply by th3ch0s3ntw0** [reply]
i cant, this is what it gives me when i try to click that
![image](https://github.com/maruohon/litematica/assets/165350050/b1ce6498-789f-402b-a4a1-52243ef53145)

**Reply by th3ch0s3ntw0** [reply]
pls help

**Reply by sakura-ryoko** [reply]
Do it from the Litematica Main Menu.
![2024-03-29_01 37 11](https://github.com/maruohon/litematica/assets/116967773/9b4bc11f-463e-4055-8f60-a1faf88d6f18)

**Reply by sakura-ryoko** [reply]
Granted, I'm not suggesting to use "Normal" mode, unless you need to; but it's just a quick and easy "work around" to make the "Simple Mode" Selection box disappear until you're ready to create an Area Selection.

**Reply by sakura-ryoko** [reply]
Another more complex method is to disable Litematica Rendering from the Config.

**Reply by th3ch0s3ntw0** [reply]
im using that menu, i go to that menu, then i click the button you have highlighted, and that is the response it gives me, i dont want to disable litematica rendering in total, because i have a schematic for a farm that im using, so disabling the rendering would disable that too, you can also just press M+R to completely disable the rendering in game, unless you mean something else?

**Reply by sakura-ryoko** [reply, marked as answer]
I've only received that message from within the "Area Editor" screen, which is because, in Normal Mode, the "Area Editor" requires a selection to be created.  Perhaps try using the "Move to Player" first, exit the "Area Editor" then use the button I highlighted from the Main Menu.

**Reply by th3ch0s3ntw0** [reply]
that worked, although switching back, its still there, i dont care between normal or simple so ill just use normal for now, but not sure why this would be happening, thank you for your help though!

**Reply by sakura-ryoko** [reply]
Then, if you want to use Simple Mode in the future, just switch it back on, and proceed as normal.

**Reply by sakura-ryoko** [reply]
Because Simple Mode automatically creates a single Selection Box, no matter if you wanted to or not, so the easiest way to just make it disappear and forget about it, is to switch to Normal Mode, and have no active Selection Boxes.

**Reply by sakura-ryoko** [reply]
Or Optionally, as you said use M + R to disable Litematica Rendering.

**Reply by th3ch0s3ntw0** [reply]
oh, simple mode ALWAYS has a box?? then theres your answer lmao, that explains it, you just cant get rid of it whatsoever??

**Reply by sakura-ryoko** [reply]
Correct.

**Reply by maruohon** [reply]
> oh, simple mode ALWAYS has a box??

Yes. In the very early versions of the mod there was no Simple mode, and a common complaint was that it's too complicated and annoying to have to go and create an area selection save file via the Area Selection Browser before you can select stuff. And as an added annoyance the early versions didn't even have the Area Editor menu yet, so you had to do all of the sub-region management purely via hotkeys...

So the Simple mode was added for any simple use cases where you just need one box, either for small simple schematics or some common creative mode tasks such as deleting or filling areas. So the Simple mode always has one area selection that is locked to exactly one sub-region box. You can't create more sub-regions, and you also can't get rid of the one it has, so it's basically always available for use while you are in the Simple mode.

So this results in the inability to get rid of that selection while in Simple mode. You basically have three different ways to "get rid of it" which here means to not see it:
1) Turn off the area selection box rendering (in Visuals -> `enableAreaSelectionBoxesRendering` or using a hotkey `toggleAreaSelectionBoxesRendering`)
2) Switch to the `Normal` selection mode and unselect any open selection in the Area Selection Browser menu
3) (if you are in `Simple` mode) just collapse it somewhere out of the way (in `Expand` corners mode by right clicking where you want to move it, or in `Corners` mode by both left and right clicking in the same spot)
